stdlib/3/importlib/metadata.pyi

```diff
@@ -0,0 +1,82 @@
+import abc
+import os
+import pathlib
+import sys
+from email.message import Message
+from importlib.abc import MetaPathFinder
+from typing import Any, Dict, Iterable, List, NamedTuple, Optional, Tuple, Union, overload
+
+if sys.version_info >= (3, 8):
+    class PackageNotFoundError(ModuleNotFoundError): ...
+    class EntryPoint(NamedTuple("EntryPointBase", [("name", str), ("value", str), ("group", str)])):
```

Comment:
We can define these with variable annotations now (thanks for doing the work to make that happen).